scripts/beachball/src/config.test.ts

```diff
@@ -91,4 +113,46 @@ describe(`beachball configs`, () => {
 
     expect(webComponentsConfig.changelog).toEqual(sharedConfig.changelog);
   });
+
+  it(`should generate tools release config`, () => {
+    expect(toolsConfig.scope).toEqual(
+      expect.arrayContaining([
+        ...excludedPackagesFromReleaseProcess,
+        'packages/eslint-plugin',
+        'scripts/api-extractor',
+        'scripts/babel',
+        'scripts/beachball',
+        'scripts/cypress',
+        'scripts/dangerjs',
+        'scripts/executors',
+        'scripts/fluentui-publish',
+        'scripts/generators',
+        'scripts/github',
+        'scripts/gulp',
+        'scripts/jest',
+        'scripts/lint-staged',
+        'scripts/monorepo',
+        'scripts/package-manager',
+        'scripts/perf-test-flamegrill',
+        'scripts/prettier',
+        'scripts/projects-test',
+        'scripts/puppeteer',
+        'scripts/storybook',
+        'scripts/tasks',
+        'scripts/test-ssr',
+        'scripts/triage-bot',
+        'scripts/ts-node',
+        'scripts/update-release-notes',
+        'scripts/utils',
+        'scripts/webpack',
+        'tools/eslint-rules',
+        'tools/workspace-plugin',
+      ]),
+    );
+
+    // Ensure that no v8/vNext packages are included in the tools release
+    expect(toolsConfig.scope.some(scope => scope.startsWith('packages/react-components'))).toBe(false);
```

Comment:
let's add some v8 package check
```suggestion
    expect(toolsConfig.scope.some(scope => scope.startsWith('packages/react-components'))).toBe(false);
    expect(toolsConfig.scope.some(scope => scope === 'packages/react')).toBe(false);
```

Reply:
`expect(toolsConfig.scope.some(scope => scope.startsWith('packages/react'))).toBe(false);` covers both v8 and vNext packages